StreamChat.xcodeproj/project.pbxproj

```diff
@@ -245,6 +246,8 @@
 		8A5B6A6124810C6B0082C023 /* Channel+KeystrokeTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A5B6A6024810C6B0082C023 /* Channel+KeystrokeTests.swift */; };
 		8A5B6A6424810E660082C023 /* ClientTestCase.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A5B6A6324810E660082C023 /* ClientTestCase.swift */; };
 		8A5D3EF924AF749200E2FE35 /* ChannelId_Tests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A5D3EF824AF749200E2FE35 /* ChannelId_Tests.swift */; };
+		8A618E4524D19D510003D83C /* WebSocketPingController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A618E4424D19D510003D83C /* WebSocketPingController.swift */; };
+		8A618E4824D2D1160003D83C /* WebSocketEventsMock.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A9BC2EB2487D9F5004C5E69 /* WebSocketEventsMock.swift */; };
```

Comment:
Why is this here???

Reply:
I added it to use `healthCheckEvent` data.